sys/clif/clif.c

```diff
@@ -323,7 +327,21 @@ ssize_t clif_get_attr(const char *input, size_t input_len, clif_attr_t *attr)
         attr->key_len = pos - attr->key;
     }
 
+    /* either the value is unquoted (0) or quoted (2) */
+    if (quotes != 0 && quotes != 2) {
+        DEBUG("Incorrect number of unescaped quotes found: %d\n", quotes);
+        goto error;
+    }
+
+    if (scan_value && !attr->value_len) {
+        DEBUG("Empty value found");
+        goto error;
+    }
+
     return pos - input;
+
+error:
+    return CLIF_NOT_FOUND;
```

Comment:
Do we really need a `goto` for just a `return`?

Reply:
See 8a0129a69bb221b931e165c68df3e26be2b6546b